zebrad/Cargo.toml

```diff
@@ -296,4 +296,4 @@ zebra-grpc = { path = "../zebra-grpc", version = "0.1.0-alpha.5" }
 zebra-utils = { path = "../zebra-utils", version = "1.0.0-beta.38" }
 
 [lints.rust]
```

Comment:
###### Optional Suggestion

```suggestion
# TODO: Remove 'cfg(zcash_unstable, values("nu6"))' once it's no longer needed for NU6.
[lints.rust]
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/8694/commits/3a730b7fcaba623179703daa6797cb886e6dc381.